src/main/java/net/openhft/chronicle/queue/impl/single/SingleChronicleQueue.java

```diff
@@ -572,16 +572,42 @@ public int nextCycle(int cycle, @NotNull TailerDirection direction) throws Parse
         return pool.nextCycle(cycle, direction);
     }
 
+    @Deprecated
```

Comment:
We should indicate "for removal" and when the method is to be removed, for example "in x.23"